src/lib/helpers.ts

```diff
@@ -249,3 +251,21 @@ export const CatchCollectionNotFoundException = (error) => {
 
   throw error
 }
+
+/**
+ * Converts all object keys to snake case.
+ * @param object — The object to convert.
+ * @return — Returns the object with converted keys.
+ */
+export const snakeCaseKeys = (
```

Comment:
I've added this helper to convert keys to snake case to not have to add them manually and avoid sending the unconverted params when using `...convertConnectionArgsToGravityArgs(options)` ([example](https://github.com/artsy/metaphysics/pull/5157/files#diff-d51c53fbca4329082a76fe9d905e961bd040f965f24579f19217ae5806cf4abbR100)).